django/template/defaulttags.py

```diff
@@ -1169,7 +1169,7 @@ def now(parser, token):
     return NowNode(format_string, asvar)
 
 
-@register.simple_tag(takes_context=True)
+@register.simple_tag(name="querystring", takes_context=True)
 def query_string(context, query_dict=None, **kwargs):
```

Comment:
Since the tests were renamed to use `querystring`, is there a reason to not rename the function as well?

Reply:
It can be renamed (I think I did this to avoid my IDE suggesting it is a typo)